v2.0.0

bergarces opened the pull request:
### Added
- Add `destroy` method to `TrezorKeyring`, which replaces `dispose` ([#179](https://github.com/MetaMask/eth-trezor-keyring/pull/179))

### Changed
- **BREAKING:** Separate the bridge from the keyring ([#143](https://github.com/MetaMask/eth-trezor-keyring/pull/143))
  - The Trezor bridge is now a separate class (`TrezorConnectBridge`), which must be constructed separately from the keyring and passed in as a constructor argument.
  - The bridge initialization has been moved from the keyring constructor to the keyring `init` method. The bridge is expected to be passed to the keyring uninitialized, and the keyring `init` method is expected to be called after keyring construction (before the keyring is used).
  - The keyring constructor no longer accepts keyring state. Instead, any pre-existing keyring state should be passed to the `deserialize` method after construction.

### Removed
- **BREAKING:** Remove `dispose` method from `TrezorKeyring`, which is replaced by `destroy` ([#179](https://github.com/MetaMask/eth-trezor-keyring/pull/179))

Gudahtt reviewed (commented):
Looks good! Though maybe we should wait on #179 first, since it's very small and also breaking.

Gudahtt reviewed (approved):
LGTM!

bergarces commented:
> Looks good! Though maybe we should wait on #179 first, since it's very small and also breaking.

I've rebased with changes from that PR and added it to the changelog (I've added at Added + Removed, but happy to do it as one entry in Changed if that's more clear).
